<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cmakeSettings</class>
 <widget class="QWidget" name="cmakeSettings">
  <property name="windowTitle">
   <string>CMake Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="KButtonGroup" name="system">
     <property name="title">
      <string>System Settings</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="cmakeBinaryLabel">
        <property name="text">
         <string>CMake executable:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="KUrlRequester" name="cmakeBinary">
        <property name="toolTip">
         <string>Path to CMake executable</string>
        </property>
        <property name="whatsThis">
         <string>Path to CMake executable</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="parentDirsLookupCnt">
        <property name="toolTip">
         <string>&lt;p&gt;How many parent directories to check for &lt;filename&gt;CMakeLists.txt&lt;/filename&gt; if the latter is absent in a current document's path&lt;/p&gt;</string>
        </property>
        <property name="maximum">
         <number>32</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="toolTip">
         <string>&lt;p&gt;How many parent directories to check for &lt;filename&gt;CMakeLists.txt&lt;/filename&gt; if the latter is absent in a current document's path&lt;/p&gt;</string>
        </property>
        <property name="text">
         <string>CMakeLists lookup parents:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Auxiliary CMake Module Directories</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QListWidget" name="moduleDirs">
        <property name="toolTip">
         <string>Auxiliary paths with CMake modules. CMake will try to parse the initial comment for every module found, so you can see a help screen for custom modules in the tool view.</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="buttons_vl">
        <item>
         <widget class="QPushButton" name="addButton">
          <property name="toolTip">
           <string>Open a file dialog to select a path to be added.</string>
          </property>
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="removeButton">
          <property name="toolTip">
           <string>Removes the currently selected path from a list of auxiliary modules.</string>
          </property>
          <property name="text">
           <string>Remove</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="vs_buttons">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="KButtonGroup" name="session">
     <property name="title">
      <string>Session Settings</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="projectBuildDirLabel">
        <property name="text">
         <string>Project build dir:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="KUrlRequester" name="projectBuildDir">
        <property name="toolTip">
         <string>Path to a project build directory associated with a current session</string>
        </property>
        <property name="whatsThis">
         <string>Path to a project build directory associated with a current session</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="vs_bottom">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KButtonGroup</class>
   <extends>QGroupBox</extends>
   <header>kbuttongroup.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
